build.sbt

```diff
@@ -67,6 +67,13 @@ lazy val publishSettings = Seq(
     </licenses>,
   sonatypeCredentialHost := "s01.oss.sonatype.org",
   sonatypeRepository := "https://s01.oss.sonatype.org/service/local",
+  publish / skip := {
```

Comment:
It seems a little weird to use `skip` as a hook to error. You may want to add a comment indicating that this is what is going on here.

Reply:
Oh yeah I meant to add a comment and totally forgot. Yeah there isn't like a "publishing hook" key so I arbitrarily picked this, definitely needs to be explained.